### PR TITLE
Adds a station trait that fills maintenance with glowsticks

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
This PR adds a station trait that fills maintenance with glowsticks (and rarely, floor lights). Thanks Smartkar for the idea. The PR was initially about something else, but people said that adding tiling to maint would ruin its style.

Here a couple examples (toned down distribution slightly) since then.
glo1
glo2

## Why It's Good For The Game
An utilitarian station trait that also mildly aesthetic.

## Changelog

:cl:
add: A station trait that fills maintenance with glowsticks.
/:cl:

### CODE REVIEW

**Comment by the-orange-cow** [comment]
nightmares seething

**Comment by Metekillot** [comment]
Make it include disco balls, floor pills, and jabroni suits, too. And lollipops

**Comment by Pumpkinoe** [comment]
> Make it include disco balls, 

god please no

**Comment by DaCoolBoss** [comment]
A station buff that gets weaker as the round progresses. Very interesting.

File: code/datums/station_traits/positive_traits.dm [hunk 1]
```diff
@@ -63,6 +63,55 @@
...
+			if(skip_this)
+				continue
+			if(prob(3.4)) ///Rare, but this is something that can survive past the lifespawn of glowsticks.
+				new /obj/machinery/light/floor(chosen)
+				continue
+			var/stick_type = pick(glowsticks)
+			var/obj/item/flashlight/glowstick/stick = new stick_type(chosen)
+			///we want a wider range, otherwise they'd all burn out in about 20 minutes.
+			stick.max_fuel = stick.fuel = rand(10 MINUTES, 45 MINUTES)
+			stick.turn_on()
```
Comment (Time-Green) [inline code/datums/station_traits/positive_traits.dm:113]: me right now

**Pull request merged**